### DISCUSSION TITLE
Run renovateBot on a project monorepo but in the logs "Repository is disabled"

### DISCUSSION THREAD

**Discussion by yuxingchen9732** [renovatebot/renovate#18729] [Request Help] [answered]
I want to run renovateBot on a project monorepo, but it isn't works,
Logs:

INFO: Repository started (repository=@@@@@)
       "renovateVersion": "34.2.0"
 INFO: Repository is disabled - skipping (repository=@@@@@)
 INFO: Repository finished (repository=@@@@@)
       "cloned": true,
       "durationMs": 4595

**Comment by viceice** [comment]
post the renovate config of the repo and provide debug logs.

**Comment by rarkins** [comment, marked as answer]
It may have been disabled server side for various reasons (e.g. non stop timeouts, or disk space, etc). What's the job ID?

**Reply by yuxingchen9732** [reply]
Thanks for your answer , the job id is #3267773835,
if I want to work with a specific branch like "test", should I create the "renovate.json" in the root of my branch "matser" and then I can work with "test"?, without the "renovate.json" in branch master, there will be error " Repository disable"

**Reply by rarkins** [reply]
You need a config file in the default branch. You can then configure `baseBranches` to tell Renovate which branches to scan

**Reply by rarkins** [reply]
Are you self-hosting or using the hosted app? I thought you were using the hosted app, but it has no job ID that high.

**Reply by yuxingchen9732** [reply]
It's self-hosting, and now I added the config file in the default branch, I think it works,
another question: where can I find the configuration if I want to reduce the noise of my projet mono repo? 

thanks a lot

**Reply by rarkins** [reply]
https://docs.renovatebot.com/noise-reduction/
